scripts/Tools/compare_test_results

```diff
@@ -66,6 +66,12 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
 
     CIME.utils.setup_standard_logging_options(parser)
 
+    parser.add_argument("-n", "--namelists-only", action="store_true",
+                        help="Only analyze namelists.")
+
+    parser.add_argument("--hist-only", action="store_true",
```

Comment:
Would it make sense to also add a short option here, '-h', analogous to the '-n' option for --namelists-only?